pkg/api/customization/setting/setting.go

```diff
@@ -11,3 +24,58 @@ func Formatter(apiContext *types.APIContext, resource *types.RawResource) {
 		resource.Values["customized"] = true
 	}
 }
+
+type Handler struct{}
+
+func (h *Handler) UpdateHandler(apiContext *types.APIContext, next types.RequestHandler) error {
```

Comment:
any reason why this is made as a part of UpdateHandler vs doing it in the setting store, Update method?

Reply:
don't want to add a new store for setting